src/store/collectives.js

```diff
@@ -373,12 +373,13 @@ export default {
 		 * @param {object} object the property object
 		 * @param {number} object.collectiveId ID of the collective to be shared
 		 * @param {number} object.pageId ID of the page to be shared
+		 * @param {number} object.password optional password for the share
```

Comment:
```suggestion
		 * @param {?string} object.password optional password for the share
```

Reply:
Good catch, fixed now. It will always be an empty string tough, just empty if no password is set.